company-profiles/shippabo.md

```diff
@@ -0,0 +1,62 @@
+# Shippabo
+
+## Company blurb
+
+Shippabo is a cloud-based supply chain management platform built by a team of shippers and supported by logistics experts. Through Shippabo, businesses can oversee their supply chain from anywhere, at any time, and on their terms.
+
+We are committed to empowering everyone along the supply chain by bringing greater transparency, collaboration, and actionable data to the world of shipping and logistics.
+
+We are a team of shippers, logistics consultants, and technologists who came together to create the change we always wanted to see in the industry. Since Shippabo was founded in 2015, we’ve built a technology platform based on our own lived experience. Shippabo makes shipping faster, more affordable, and more transparent — so companies can ship on their terms.
+
+## Company size
+
+We are a company of 20-50 employees all working remotely from the various states in the US and provinces in Canada!
+
+## Remote status
+
+💻 Remote: This is a fully remote position. We are in 10+ US states, Canada and China and are used to working on different time zones. We are big fans of trust, autonomy and asynchronous communications!
```

Comment:
China is only mentioned in this sentence but nowhere else, can you clarify please?

Reply:
Yes we have a logistic operations office in China, but are currently only hiring for North America at the moment